node/src/Pos/Delegation/Cede/Logic.hs

```diff
@@ -103,3 +111,115 @@ dlgReachesIssuance i d psk = reach i
         Just psk'@ProxySecretKey{..}
             | pskDelegatePk == d -> pure $ psk' == psk
             | otherwise          -> reach pskDelegatePk
+
+
+-- | Verifies a header from delegation perspective (signature checks).
+dlgVerifyHeader ::
+       (MonadCedeRead m)
+    => MainBlockHeader ssc
+    -> ExceptT Text m ()
+dlgVerifyHeader h = do
+    -- Issuer didn't delegate the right to issue to elseone.
+    let issuer = h ^. mainHeaderLeaderKey
+    let sig = h ^. gbhConsensus ^. mcdSignature
```

Comment:
@neongreen said _mda_ when I used two `^.`.

Reply:
That's true. @volhovM _mda_